### DISCUSSION TITLE
Issue requesting PCAP

### DISCUSSION THREAD

**Discussion by haleinfosec** [Security-Onion-Solutions/securityonion#2843] [Unsupported Versions]
I am having trouble pulling PCAP files in Standalone v2.3.1 environment. I have verified that Elasticsearch is up and running, but seem to either get a "Elasticsearch document was not found" message or the message below. How can I fix this? Thank you!

![2021-02-03 10_41_28-Security Onion](https://user-images.githubusercontent.com/74373700/106770915-6e3c0500-660c-11eb-958f-152ef5aa8ad0.png)

**Comment by weslambert** [comment]
Is this from Hunt/Alerts or Kibana? Does it happen when attempting to pull PCAP from both interfaces?
